src/components/Organisms/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useRef, useState, useEffect } from "react";
+import styled from "styled-components";
+import FONTFAMILY from "../../variables/font_family";
+import TEXTS from "../../variables/texts";
+import COLOR from "../../variables/color";
+import AddTaskButton from "../Atoms/AddTaskButton/index";
+import Tasks from "../Molecules/index";
+
+const TodoCard = () => {
+  const [taskList, setTaskList] = useState([{ name: "", initializing: true }]);
+
+  const onAddTaskButtonClick = () => {
+    setTaskList([...taskList, { name: "", initializing: true }]);
+  };
+
+  const onTaskComplete = ({ index }) => {
+    const deleteList = [...taskList];
+    const listIndex = index;
+    setTaskList(
+      deleteList.filter((value, index, list) => {
+        return index !== listIndex;
+      })
```

Comment:
```suggestion
    const listIndex = index;
    setTaskList(
      taskList.filter((value, index, list) => {
        return index !== listIndex;
      })
```
deleteList と taskList は同じなので、これでもよさそうです